apps/web/src/components/honors-list/index.tsx

```diff
@@ -0,0 +1,77 @@
+import type { AwardedHonor, Honor } from "@tietokilta/cms-types/payload";
+import { ChevronDownIcon } from "@tietokilta/ui";
+import _ from "lodash";
+
+function AwardedPersonDropdown({
+  awardedPerson,
+}: {
+  awardedPerson: AwardedHonor;
+}) {
+  return (
+    <div className="not-prose shadow-solid relative my-4 flex overflow-hidden rounded-md border-2 border-gray-900 px-2 pt-11 font-mono md:px-3">
+      <details className="group contents">
+        <summary
+          className={`absolute left-0 top-0 flex w-full ${awardedPerson.description ? "cursor-pointer" : ""} justify-between border-b-2 border-gray-900 bg-gray-100 p-2 md:px-3 [&::-webkit-details-marker]:hidden [&::marker]:hidden`}
```

Comment:
Use the `cn` helper for conditional styles in tailwind so it's more robust (handles conflicting styles, order, merging etc. automatically and makes sure the styles are generated)
```suggestion
          className={cn("absolute left-0 top-0 flex w-full justify-between border-b-2 border-gray-900 bg-gray-100 p-2 md:px-3 [&::-webkit-details-marker]:hidden [&::marker]:hidden", !!awardedPerson.description && "cursor-pointer"}
```
_(also needs the import and re-formatting so these changes should be applied locally)_